backend/src/scripts/utility/users/identifyInactiveUsers.js

```diff
@@ -0,0 +1,91 @@
+/**
+ *  Copyright (C) 2025 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const Path = require('path');
+const { v5Path } = require('../../../interop');
+const FS = require('fs');
+const DBHandler = require('../../../v5/handler/db');
+
+const { logger } = require(`${v5Path}/utils/logger`);
+const { getUsersByQuery } = require(`${v5Path}/models/users`);
+const { getLastLoginDate } = require(`${v5Path}/models/loginRecords`);
+
+const DEFAULT_OUT_FILE = 'inactiveUsers.csv';
+
+const writeResultsToFile = (results, outFile) => new Promise((resolve) => {
+	logger.logInfo(`Writing results to ${outFile}`);
+	const writeStream = FS.createWriteStream(outFile);
+	writeStream.write('Username,First Name,Last Name,Email,Company,Last Login\n');
+	results.forEach(({ user, firstName, lastName, email, company, lastLogin }) => {
+		writeStream.write(`${user},${firstName},${lastName},${email},${company},${lastLogin}\n`);
```

Comment:
Can we format the lastLogin to just DD/MM/YY?

You'll find examples from other scripts 
![image](https://github.com/user-attachments/assets/07003455-876a-4f34-845c-b2aff2896232)